src/app/legacy/containers/Navigation/index.test.jsx

```diff
@@ -113,4 +115,70 @@ describe('Navigation Container', () => {
       expect(href).toEqual(navItem.url);
     });
   });
+
+  describe('Navigation Event Tracking', () => {
+    const scrollEventTrackingData = {
+      componentName: 'scrollable-navigation',
+    };
+
+    const dropdownEventTrackingData = {
+      componentName: 'dropdown-navigation',
+    };
+
+    it('should call the view tracking hook with the correct params when on scrollable navigation', () => {
```

Comment:
Is the only param that it is checking the componentName?

(first time looking at these so I might have missing knowledge here)

Reply:
Me and @holchris had a discussion looking into this and have some questions about click tracking tests:

These tests here are like the radio schedule ones so they have been done in this way before, and are approved and passing. The tests for the click tracking on the message banner are a bit different as they include actually clicking the link before asserting the click tracking spy has been called.

1. Do we need to simulate the click before asserting that the clickTrackerSpy was called?
2. Why does the test pass when we haven't done a click? (both these ones and the radio schedule ones)
3. Why does the radio schedule not have a click before checking the clickTrackerSpy?

Reply:
Yeah you should simulate the click and probably check `sendEventBeacon`, or use something to check something has fired.

It passes because the click and view tracker hooks are called with the tracking data, they're just sitting their waiting to fire off the callback when a click is made.

Reply:
You can use the testing library's `user-event` to simulate clicks https://testing-library.com/docs/user-event/v13/#clickelement-eventinit-options